webapp/package.json

```diff
@@ -20,6 +20,7 @@
     "intl": "1.2.5",
     "jasny-bootstrap": "3.1.3",
     "jquery": "3.2.1",
+    "key-mirror": "1.0.1",
```

Comment:
mattermost-redux uses it from npm now and webpack was throwing a fit because we had a non npm version conflicting with the npm version (both were named key-mirror)